nes-common/include/Util/Strings.hpp

```diff
@@ -147,4 +147,8 @@ std::vector<T> splitWithStringDelimiter(std::string_view inputString, std::strin
         | std::views::transform([](auto optional) { return *optional; }) | std::ranges::to<std::vector>();
 }
 
+/// Converts a snake_case string to camelCase.
+/// Throws InvalidIdentifier if non-ASCII characters are encountered during case conversion.
```

Comment:
I know this is my code, but the comment regarding throwing an exception is not true.
It should state that this only supports ASCII strings.